samples/openapi3/client/3_0_3_unit_test/python/unit_test_api/paths/request_body_post_additionalproperties_allows_a_schema_which_should_validate_request_body/post.py

```diff
@@ -59,17 +59,50 @@ class ApiResponseFor200(api_client.ApiResponse):
 
 class BaseApi(api_client.Api):
 
+    @typing.overload
     def _post_additionalproperties_allows_a_schema_which_should_validate_request_body_oapg(
-        self: api_client.Api,
-        body: typing.Union[SchemaForRequestBodyApplicationJson, ],
+        self,
+        body: typing.Union[SchemaForRequestBodyApplicationJson,],
+        content_type: str = 'application/json',
+        stream: bool = False,
+        timeout: typing.Optional[typing.Union[int, typing.Tuple]] = None,
+        skip_deserialization: typing_extensions.Literal[False] = False,
+    ) -> typing.Union[ApiResponseFor200,api_client.ApiResponse,]:
+        ...
+
+    @typing.overload
+    def _post_additionalproperties_allows_a_schema_which_should_validate_request_body_oapg(
+        self,
+        body: typing.Union[SchemaForRequestBodyApplicationJson,],
+        content_type: str = 'application/json',
+        stream: bool = False,
+        timeout: typing.Optional[typing.Union[int, typing.Tuple]] = None,
+        skip_deserialization: typing_extensions.Literal[True] = True,
+    ) -> api_client.ApiResponseWithoutDeserialization:
+        ...
+
+    @typing.overload
+    def _post_additionalproperties_allows_a_schema_which_should_validate_request_body_oapg(
+        self,
+        body: typing.Union[SchemaForRequestBodyApplicationJson,],
         content_type: str = 'application/json',
         stream: bool = False,
         timeout: typing.Optional[typing.Union[int, typing.Tuple]] = None,
         skip_deserialization: bool = False,
     ) -> typing.Union[
         ApiResponseFor200,
-        api_client.ApiResponseWithoutDeserialization
+        api_client.ApiResponseWithoutDeserialization,
     ]:
+        ...
+
+    def _post_additionalproperties_allows_a_schema_which_should_validate_request_body_oapg(
+        self,
+        body,
+        content_type = 'application/json',
+        stream = False,
+        timeout = None,
+        skip_deserialization = False,
```

Comment:
Can you please keep the type hints on the inputs on the non-overloaded function?

Reply:
I will add it back.